### ISSUE TITLE
F-Droid inclusion

### ISSUE THREAD

**Issue by shuvashish76** [Moderpach/Extinguish#10]
Hello, would you consider publishing the app on F-Droid please?
See : https://gitlab.com/fdroid/fdroiddata/blob/master/CONTRIBUTING.md#merge-requests

**Comment by PeterKW** [comment]
https://apt.izzysoft.de/fdroid/index/apk/own.moderpach.extinguish

**Comment by shuvashish76** [comment]
Yes I noticed that but IzzyOnDroid is third-party F-Droid. My request is for the inclusion in official F-Droid repo. That way it'll reach out to more larger users.
(The app can co-exist on both the repositories)

**Comment by Moderpach** [comment]
I will
